bangazonreports/views/completed_orders.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+from django.shortcuts import render
+from .connection import Connection
+from bangazonapi.models import Order
+
+
+def completed_order_list(request):
+    if request.method == "GET":
+        with sqlite3.connect(Connection.db_path) as conn:
+            conn.row_factory = sqlite3.Row
+            db_cursor = conn.cursor()
+            db_cursor.execute("""
+                SELECT ord.id as OrderId, pay.merchant_name as PaymentType, SUM(prod.price) as OrderTotal, user.first_name || " " || user.last_name as FullName
+                FROM bangazonapi_order as ord, bangazonapi_payment as pay, 
+                        bangazonapi_orderproduct as ordprod, bangazonapi_product as prod,
+                        auth_user as user
+                WHERE ord.payment_type_id = pay.id AND ord.id = ordprod.order_id
+                    AND ordprod.product_id = prod.id AND ord.customer_id = user.id
```

Comment:
The customer_id on the order doesn't match up with the user.id since that field on the Order is a foreign key to the customer table